docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -77,6 +77,10 @@ If the [browsers](configuration-file.md#browsers) and [src](configuration-file.m
 > Important! Make sure to keep the browser tab that is running tests active. Do not minimize the browser window.
 > Inactive tabs and minimized browser windows switch to a lower resource consumption mode
 > where tests do not always execute correctly.
+>
+> Do not adjust page zoom during tests. Stable functionality on zoomed in our out pages is not guaranteed.
+
+During a test the TestCafe prevents native dialogs from being shown, so they don't block the page during the test run. Learn how to handle native dialogs [here](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html#handle-native-dialogs).
```

Comment:
We can say shorter:

`During a test, TestCafe suppresses native dialogs so they...`

Reply:
`TestCafe` does not require an article